config/initializers/swagger_server_url.rb

```diff
@@ -0,0 +1,6 @@
+return if ENV["DOMAIN"].blank?
+
+Dir[Rails.root.join("public/api/docs/**/swagger.yaml")].each do |swagger_file|
+  swagger_doc = File.read(swagger_file)
+  File.write(swagger_file, swagger_doc.gsub("http://0.0.0.0:3000", ENV["DOMAIN"]))
```

Comment:
should this include `https//:`? as that domain doesn't? Also not sure if this is what is affecting the authorisation and trying it out in the actual docs, as we get a url like this:
```
'https://npq-registration-review-1891-web.test.teacherservices.cloud/api/docs/v3/cpd-npq-review-1891-web.test.teacherservices.cloud/api/v3/npq-applications?cohort=2021
```

Reply:
Also `DOMAIN` is being set as the internal domain in the review app? `https://cpd-npq-review-1891-web.test.teacherservices.cloud`

So seeing an error:

```
Failed to fetch.
Possible Reasons: CORS
Network Failure URL scheme must be "http" or "https" for CORS request.
```